dips/dip-10.md

```diff
@@ -178,14 +181,24 @@ The format of the success response is:
 | status       | str | Y | Either `success` or `failure`. |
 | result       | Result object | Y | The Result obejct of response. |
 | cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
+| error          | [OffChainErrorObject](https://github.com/diem/dip/blob/main/dips/dip-1.mdx#offchainerrorobject) | N | Details of the error when status == "failure" |
+
 
 **CommandResultObject:**
 
 | Field 	    | Type 	     | Required? 	| Description 	           |
 |-------	    |------	     |-----------	|-------------	           |
 | _ObjectType   | str        | Y | Fixed value: `ReferenceIDCommandResponse`|
-| recipient_address       | str | Y | Receiver's on-chain address |
+| receiver_address       | str | Y | Receiver's [bech32 identifier](https://github.com/diem/dip/blob/main/dips/dip-5.md) |
```

Comment:
May consider use "account identifier" instead of bech32 identifier, because it is defined in dip-5